dymos/load_case.py

```diff
@@ -149,6 +150,12 @@ def load_case(problem, previous_solution):
             if init_val_path:
                 problem.set_val(init_val_path[0], prev_state_val[0, ...], units=prev_state_units)
 
+            if options['fix_final']:
```

Comment:
It's better to use the OpenMDAO helper `issue_warning` in instead of warnings.warn.  See https://github.com/OpenMDAO/dymos/blob/0903ae1eb6aaa6eb2508c72202c14319d3a9cf37/dymos/transcriptions/pseudospectral/radau_pseudospectral.py#L416